config/install/field.field.paragraph.stat.field_stat_value.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_stat_value
+    - paragraphs.paragraphs_type.stat
+id: paragraph.stat.field_stat_value
+field_name: field_stat_value
+entity_type: paragraph
+bundle: stat
+label: Value
+description: ''
+required: true
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings: {  }
+field_type: string
```

Comment:
Should this be a number field if it has to be numeric to scroll?

Reply:
I went with a string to allow for flexibility in what it displayed. The animation style is option (via checkbox) and the javascript includes a check for it being a number anyway.

Reply:
I realised while I was out for a walk that this might not be a number and that this comment was irrelevant, haha.